plugins/woo-ai/src/utils/productData.ts

```diff
@@ -36,6 +37,7 @@ const getTags = (): string[] => {
 	return tags.filter( ( tag ) => tag !== '' );
 };
 
+// TODO: Not working in my testing when manually adding attributes.
```

Comment:
Can we just delete this, then?

Reply:
I _believe_ these may work in other contexts, such as if you're editing an existing product, so we may just want to improve them. @nima-karimi could clarify more here.

Reply:
hmm. Could you share your test steps? If you press the "Save Attributes" button then it should include those attributes in the request 🤔

Reply:
Thanks @nima-karimi , it looks like the categories selectors only worked when those categories had been applied on the last page refresh, and not recently added/checked. 

For the attributes, it looked like it was only working for global attributes, and not local I believe. I've fixed those both in the last couple commits ([1](https://github.com/woocommerce/woocommerce/pull/38930/commits/81f15148873ee4443dc8d6e4d0481317c2e63cff), [2](https://github.com/woocommerce/woocommerce/pull/38930/commits/e370aba1d52c26791cf9ee0dc0951840a3c2ef08))